split.Console/Models/Game.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace split.Console.Models
+{
+    public class Game
+    {
+        public Guid Id;
+        public string Code;
+        public GameType Type;
+        public List<Player> Players;
+        public GameState State;
+
+        public Game(GameType type)
+        {
+            Code = GenerateRoomCode();
+            Id = new Guid(Code);
```

Comment:
I guess you could create the long GUID and just take first or last four characters? Or [pick however many members](https://msdn.microsoft.com/en-us/library/system.guid.tobytearray(v=vs.110).aspx) of a ```ByteArray``` you need?